crane/views/v2.py

```diff
@@ -63,6 +65,12 @@ def name_redirect(relative_path):
     if not base_url.endswith('/'):
         base_url += '/'
 
+    # perform base_url rewrite if relevant CDN config present
+    url_match = current_app.config.get(config.SECTION_CDN, {}).get(config.KEY_URL_MATCH)
+    url_replace = current_app.config.get(config.SECTION_CDN, {}).get(config.KEY_URL_REPLACE)
+    if url_match and url_replace:
```

Comment:
this replacement can be done after https://github.com/pulp/crane/pull/88/files#diff-3421ac601282979580dfcb0aa16403ebR115 if it is the case